### DISCUSSION TITLE
IDEA安装插件一直卡在等待页面问题

### DISCUSSION THREAD

**Discussion by dgq666** [community/community#18719] [Other Feature Feedback, Questions, & Ideas]
安装插件，重启IDEA，下一步登录git进行授权，网页上完成授权后，IDEA页面一直卡在waiting页面 
![image](https://user-images.githubusercontent.com/48819277/174026064-39d59995-da29-4c1b-8301-dfe8e8e1b61e.png)

**Comment by Ventivu** [comment]
That is a unbelieveable error for me too,I don't know why I finished all step but it don't do anything

**Comment by h742004725** [comment]
新版本的bug，下载旧版本的就可以了

**Comment by dgq666** [comment]
持续操作了2个小时  中午搞好了 也不知道为啥

**Reply by dgq666** [reply]
就一直重试 重试
